src/postsList/usePostsList.js

```diff
@@ -1,21 +1,21 @@
 import { useEffect, useState } from "react";
+import { api } from "../shared/api.js";
+
+const { fetchUsers, fetchPosts, fetchPhotos, fetchAlbums } = api;
```

Comment:
The whole point of exporting an entire API object is to use it like this: `api.fetchUsers()`

So this destructuring is not a good idea

Reply:
done, got it now...